CHANGELOG.md

```diff
@@ -16,7 +21,21 @@
 - `ChatThreadClient` `remove(participant)` method now accepts `CommunicationIdentifier` instead of a string
 - For `Participant` renamed `user` property to `id`
 
+**Azure Communication Calling**
+- `Renderer` renamed to `VideoStreamRenderer`.
+- `AudioDeviceInfo` removed from `DeviceManager`, please use iOS system API's in your application to switch between audio devices.
```

Comment:
Should this be considered a breaking change?

Reply:
Yes, as this drop will break previous contract. Code change is needed on User end in order to consume this if the previous API was used.